src/devtools/client/webconsole/reducers/messages.ts

```diff
@@ -259,12 +262,6 @@ const messagesSlice = createSlice({
 
       return removeMessagesFromState(state as MessageState, removedIds);
     },
-    //  This is only here to force recalculation after filters are updated
-    // TODO Find a way to rework `visibleMessages` as derived data and remove this
-    filterStateUpdated(state, action: PayloadAction<FiltersState>) {},
```

Comment:
:speech_balloon: Yeah, this was a holdover from when I was doing some early RTK reducer migration.  I had to temporarily dispatch an extra action to keep this reducer in sync with the `filtersReducer`, and eventually resolved it by moving the filters state inside of this slice.  Looks like I just forgot to remove `filterStateUpdated`, so good catch!